components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -313,6 +315,61 @@ struct SwapResponse {
   // Note we could also expose "sources" later
 };
 
+// Structs to model Jupiter (swap) HTTP API interactions
+// Swagger spec: https://quote-api.jup.ag/docs/static/index.html
+//
+// [XXX] - amount field is incorrectly modelled as number in spec.
```

Comment:
It's a comment label supported by Jetbrains IDEs when it's neither of `FIXME` or `TODO`. I changed the wording to start with a `Note..` to avoid confusion. See 029b63746e4d1cc30d65d58a706e6fb47c482437.